### ISSUE TITLE
新增一个导航

### ISSUE THREAD

**Issue by tomnattle** [swaggo/swag#624]
var importJs=document.createElement('script') ;
importJs.setAttribute("type","text/javascript") ;
importJs.setAttribute("src", 'http://ajax.microsoft.com/ajax/jquery/jquery-1.4.min.js');
document.getElementsByTagName("head")[0].appendChild(importJs) ;

var creatediv= function(){
    var parentdiv=$('<div style="position:fixed;right:30px;top:60px;width:250px; padding:10px; background:#fff; border:solid 1px #666;"></div>');
    parentdiv.attr('id','parent');
    parentdiv.appendTo('body');
}

var createItem= function(id, name){
    var parentdiv=$('<div style="width:100px;float:left"><a style="color:grey" href="#'+id+'">'+name+'</a></div>');
    parentdiv.attr('id','parent');
    parentdiv.appendTo('#parent');
}
setTimeout(function (){
creatediv();
$(".opblock-tag .nostyle span").each((i,o)=>{
	createItem($(o).parent().parent().attr("id"),$(o).text())
});
}, 3000);

**Comment by tomnattle** [comment]
在chrome的 console中执行，浏览器来方便一些。

**Comment by sdghchj** [comment]
是不是跑错片场了？
